client/http_client.go

```diff
@@ -154,21 +154,12 @@ func (slc *HttpClient) scheme() string {
 }
 
 func (slc *HttpClient) makeHttpRequest(url string, requestType string, requestBody *bytes.Buffer) ([]byte, int, error) {
-	req, err := http.NewRequest(requestType, url, requestBody)
-	if err != nil {
-		return nil, 0, err
-	}
-
-	bs, err := httputil.DumpRequest(req, true)
-	if err != nil {
-		return nil, 0, err
-	}
+	var (
+		resp *http.Response
+		bs   []byte
+	)
+	body, _ := ioutil.ReadAll(requestBody)
```

Comment:
Why do you silently ignore error returned in `ioutil.ReadAll(...)`? Please check that error and no need to have a `var ( ... )` for the local variables. Just define them one per line.